src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java

```diff
@@ -152,9 +162,55 @@ public ResponseEntity<Void> createRecord(@RequestBody PatientRecord record) {
         return new ResponseEntity<>(headers, HttpStatus.CREATED);
     }
 
-    @PostMapping(value = "/import", consumes = MediaType.APPLICATION_JSON_VALUE)
-    public RecordImportResult importRecords(@RequestBody List<PatientRecord> records,
+    @PostMapping(value = "/import/json", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```

Comment:
In my perspective, dividing it into different endpoints has an advantage because if I need to add support for a new file type, I can just add a new endpoint without changing the logic in the existing functions. 

I will try to use one endpoint, /import, and create private functions for different file types.

Reply:
I already created the issue, but it is **not a priority now** -- https://github.com/kbss-cvut/record-manager-ui/issues/190.